specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-04-01/examples/BackupResourceEncryptionConfig_Put.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "test-rg",
+    "vaultName": "source-rsv",
+    "api-version": "2021-01-01",
+    "parameters": {
+      "properties": {
```

Comment:
>"properties":  [](http://example.com/codeflow?start=6&length=14)

Can userAssignedIdentity and useSystemAssignedIdentity be set by user? Can we update the example?

Reply:
Made a fix. User is not supposed to set the fields using this API. The information will only be received as a part of GET call